python/pyspark/sql/context.py

```diff
@@ -93,6 +93,10 @@ def _ssql_ctx(self):
         """
         return self._jsqlContext
 
+    def conf(self):
```

Comment:
Let's make this `_conf` with a property.

Reply:
Thanks, done in 4fc0ae4.